Using ScatterPlotCanvas raises an exception and does not display

**Describe/explain the bug**

Using ScatterPlotCanvas (@0.62.0) raises an exception and does not display.

**To Reproduce**

Steps to reproduce the behavior:
1. Go to https://nivo.rocks/scatterplot/canvas/

**Screenshots**

![nivo rocks_scatterplot_canvas_](https://user-images.githubusercontent.com/1730234/86332944-2de76380-bc86-11ea-88de-f80d3eedf2d2.png)
![Screenshot from 2020-07-02 17-05-22](https://user-images.githubusercontent.com/1730234/86333054-5a9b7b00-bc86-11ea-8f83-115a3a55413e.png)

<details>

```log
react-dom.production.min.js:4454 Error: Invalid layer: annotations
    at nivo-scatterplot.esm.js:869
    at Array.forEach (<anonymous>)
    at nivo-scatterplot.esm.js:816
    at ha (react-dom.production.min.js:4485)
    at Wa (react-dom.production.min.js:5094)
    at Va (react-dom.production.min.js:5113)
    at Object.enqueueSetState (react-dom.production.min.js:2834)
    at i.BdB7.w.setState (react.production.min.js:96)
    at index.esm.js:116
da @ commons-6de9021bb955c5a6c6b6.js:1
Ea.i.componentDidCatch.n.callback @ commons-6de9021bb955c5a6c6b6.js:1
oa @ commons-6de9021bb955c5a6c6b6.js:1
ra @ commons-6de9021bb955c5a6c6b6.js:1
Ha @ commons-6de9021bb955c5a6c6b6.js:1
qa @ commons-6de9021bb955c5a6c6b6.js:1
(anonymous) @ commons-6de9021bb955c5a6c6b6.js:1
UdKW.t.unstable_runWithPriority @ commons-6de9021bb955c5a6c6b6.js:1
Nu @ commons-6de9021bb955c5a6c6b6.js:1
Iu @ commons-6de9021bb955c5a6c6b6.js:1
Pu @ commons-6de9021bb955c5a6c6b6.js:1
Cu @ commons-6de9021bb955c5a6c6b6.js:1
Wa @ commons-6de9021bb955c5a6c6b6.js:1
Va @ commons-6de9021bb955c5a6c6b6.js:1
enqueueSetState @ commons-6de9021bb955c5a6c6b6.js:1
BdB7.w.setState @ commons-6de9021bb955c5a6c6b6.js:1
(anonymous) @ commons-6de9021bb955c5a6c6b6.js:1
requestAnimationFrame (async)
t.measure @ commons-6de9021bb955c5a6c6b6.js:1
react-dom.production.min.js:4454 Error: Invalid layer: annotations
    at nivo-scatterplot.esm.js:869
    at Array.forEach (<anonymous>)
    at nivo-scatterplot.esm.js:816
    at ha (react-dom.production.min.js:4485)
    at Wa (react-dom.production.min.js:5094)
    at Va (react-dom.production.min.js:5113)
    at Object.enqueueSetState (react-dom.production.min.js:2834)
    at i.BdB7.w.setState (react.production.min.js:96)
    at index.esm.js:116
da @ commons-6de9021bb955c5a6c6b6.js:1
n.callback @ commons-6de9021bb955c5a6c6b6.js:1
oa @ commons-6de9021bb955c5a6c6b6.js:1
ra @ commons-6de9021bb955c5a6c6b6.js:1
Ha @ commons-6de9021bb955c5a6c6b6.js:1
qa @ commons-6de9021bb955c5a6c6b6.js:1
(anonymous) @ commons-6de9021bb955c5a6c6b6.js:1
UdKW.t.unstable_runWithPriority @ commons-6de9021bb955c5a6c6b6.js:1
Nu @ commons-6de9021bb955c5a6c6b6.js:1
Iu @ commons-6de9021bb955c5a6c6b6.js:1
Pu @ commons-6de9021bb955c5a6c6b6.js:1
Cu @ commons-6de9021bb955c5a6c6b6.js:1
Wa @ commons-6de9021bb955c5a6c6b6.js:1
Va @ commons-6de9021bb955c5a6c6b6.js:1
enqueueSetState @ commons-6de9021bb955c5a6c6b6.js:1
BdB7.w.setState @ commons-6de9021bb955c5a6c6b6.js:1
(anonymous) @ commons-6de9021bb955c5a6c6b6.js:1
requestAnimationFrame (async)
t.measure @ commons-6de9021bb955c5a6c6b6.js:1
react-dom.production.min.js:5986 Uncaught Error: Invalid layer: annotations
    at nivo-scatterplot.esm.js:869
    at Array.forEach (<anonymous>)
    at nivo-scatterplot.esm.js:816
    at ha (react-dom.production.min.js:4485)
    at Wa (react-dom.production.min.js:5094)
    at Va (react-dom.production.min.js:5113)
    at Object.enqueueSetState (react-dom.production.min.js:2834)
    at i.BdB7.w.setState (react.production.min.js:96)
    at index.esm.js:116
```

</details>

**Desktop**
 - OS: Ubuntu 20.04 LTS
 - Browser: Google Chrome 83.0.4103.116

--- a/packages/scatterplot/src/ScatterPlotCanvas.js
+++ b/packages/scatterplot/src/ScatterPlotCanvas.js
@@ -14,12 +14,13 @@ import {
     getRelativeCursor,
     isCursorInRect,
 } from '@nivo/core'
+import { renderAnnotationsToCanvas } from '@nivo/annotations'
 import { renderAxesToCanvas, renderGridLinesToCanvas } from '@nivo/axes'
 import { renderLegendToCanvas } from '@nivo/legends'
 import { useTooltip } from '@nivo/tooltip'
 import { useVoronoiMesh, renderVoronoiToCanvas, renderVoronoiCellToCanvas } from '@nivo/voronoi'
 import { ScatterPlotCanvasPropTypes, ScatterPlotCanvasDefaultProps } from './props'
-import { useScatterPlot } from './hooks'
+import { useScatterPlot, useScatterPlotAnnotations } from './hooks'
 
 const ScatterPlotCanvas = props => {
     const {
@@ -49,6 +50,8 @@ const ScatterPlotCanvas = props => {
         axisRight,
         axisBottom,
         axisLeft,
+
+        annotations,
 
         isInteractive,
         debugMesh,
@@ -82,6 +85,8 @@ const ScatterPlotCanvas = props => {
         nodeSize,
         colors,
     })
+
+    const boundAnnotations = useScatterPlotAnnotations(nodes, annotations)
 
     const { delaunay, voronoi } = useVoronoiMesh({
         points: nodes,
@@ -139,6 +144,8 @@ const ScatterPlotCanvas = props => {
                         axis: 'y',
                         values: gridYValues,
                     })
+            } else if (layer === 'annotations') {
+                renderAnnotationsToCanvas(ctx, { annotations: boundAnnotations, theme })
             } else if (layer === 'axes') {
                 renderAxesToCanvas(ctx, {
                     xScale,

